src/Constants.js

```diff
@@ -1,42 +1,31 @@
 "use es6";
 
-export const VERSION = "0.0.1";
```

Comment:
I first deleted `VERSION` in favor of using `import { getVersion } from './packageInfo';`, but I think deleting it would technically count as a breaking change in case a calling app is using it. I have instead refactored `getVersion` into this constant.